MBBSEmu/HostProcess/GlobalRoutines/PageUserGlobal.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using MBBSEmu.Extensions;
+using MBBSEmu.Memory;
+using MBBSEmu.Module;
+using MBBSEmu.Session;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using MBBSEmu.Session.Enums;
+
+namespace MBBSEmu.HostProcess.GlobalRoutines
+{
+    public class PageUserGlobal : IGlobalRoutine
+    {
+        public bool ProcessCommand(byte[] command, ushort channelNumber, PointerDictionary<SessionBase> sessions, Dictionary<string, MbbsModule> modules)
+        {
+            var commandString = Encoding.ASCII.GetString(command).TrimEnd('\0');
+            
+            //Check minimum length for a /p command else exit
+            if(commandString.Length < 6) return false;
```

Comment:
Move this before `commandString` is declared and have it evaluate the length of `command`. Because globals are checked on every user input, just want to make it as performant as possible.

Reply:
Fixed @enusbaum